lua/lspconfig/server_configurations/openedge_ls.lua

```diff
@@ -2,6 +2,15 @@ local util = require 'lspconfig.util'
 
 return {
   default_config = {
+    cmd = {
```

Comment:
does there have any relate standard doc ? post it in comment or in default doc will be useful.